### ISSUE TITLE
Google docs: Braille Focus issues with bulletted lists

### ISSUE THREAD

**Issue by ainekc** [nvaccess/nvda#13331]

**Comment by Adriani90** [comment]
@ainekc are you still having this issue with NVDA 2024.1 Beta 9 or last alpha version?
Also please make sure you have enabled both screen reader support (ctrl+alt+z) and braille support (ctrl+alt+h) in Google docs when the document is open.

**Comment by ainekc** [comment]
Yeah, Braille and screen reader support are enabled, I'm using a Mantis and panning, just tried Edge and chrome, still jumping eratically

**Comment by seanbudd** [comment]
Is this still an issue you are experiencing?

**Comment by seanbudd** [comment]
Closing as we haven't heard if this can still be reproduced
